extensions/wikia/UserLogin/UserSignupSpecialController.class.php

```diff
@@ -122,54 +125,61 @@ public function signupForm() {
 		// process signup
 		$redirected = $this->request->getVal( 'redirected', '' );
 		if ( $this->wg->Request->wasPosted() && empty( $redirected ) ) {
+			$this->handleSignupFormSubmit();
+		} else {
+			$this->track( 'signup-start' );
+		}
+	}
 
```

Comment:
Moved the following code down into the ```handleSignupFormSubmit``` function below.  Additional changes to not set 'msg' and 'errParam' unless result is not ```ok```, and to set them via ```this->response->setData```.

Also, added a tracking call after we've successfully created a user.